src/App.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+const todoList = [
+  {
+    id:"1",
+    title:"complete assignment",
+  },
+  {
+    id:"2",
+    title:"pull request",
+  },
+  {
+    id:"3",
+    title:"submit assignment",
+
+  }
+]
+function App() {
+  return (
+    <div>
+      <h1>Todo List</h1> 
+      <ul>
+        {
+          todoList.map(function(item){
+            return <li key={item.id}>{item.title})</li>;
```

Comment:
extra closing parentheses after item.title while will add a closing parentheses to the end of each item mapped